src/main/java/de/plushnikov/intellij/plugin/LombokPluginApplicationComponent.java

```diff
@@ -63,6 +68,25 @@ public void initComponent() {
     if (updated) {
       settings.setVersion(Version.PLUGIN_VERSION);
     }
+
+    final BuildNumber currentBuild = ApplicationInfo.getInstance().getBuild();
+    if(currentBuild.compareTo(BuildNumber.fromString("146.1154")) < 0) {
+      LOG.info("Starting injection of IntelliJ-Patch");
+      injectAgent();
+    }
+  }
+
+  private void injectAgent() {
+    LOG.info("pre injection");
+    String rootOfClass = ClassRootFinder.findClassRootOfClass(IdeaPatcher.class).substring(1);
```

Comment:
why are you removing initial slash? This is partialy responsible for test failures on *nix I believe.
What is the output for you on Windows?